proto/src/determined/api/v1/experiment.proto

```diff
@@ -424,35 +424,20 @@ message PatchExperimentResponse {
 
 // Get a list of checkpoints for an experiment.
 message GetExperimentCheckpointsRequest {
-  // Sorts checkpoints by the given field.
-  enum SortBy {
-    // Returns checkpoints in an unsorted list.
-    SORT_BY_UNSPECIFIED = 0;
-    // Returns checkpoints sorted by UUID.
-    SORT_BY_UUID = 1;
-    // Returns checkpoints sorted by trial id.
-    SORT_BY_TRIAL_ID = 2;
-    // Returns checkpoints sorted by batch number.
-    SORT_BY_BATCH_NUMBER = 3;
-    // Returns checkpoints sorted by end time.
-    SORT_BY_END_TIME = 4;
-    // Returns checkpoints sorted by state.
-    SORT_BY_STATE = 5;
-    // Returns checkpoints sorted by the experiment's `searcher.metric`
-    // configuration setting.
-    SORT_BY_SEARCHER_METRIC = 6;
-  }
   // The experiment id.
   int32 id = 1;
   // Sort checkpoints by the given field.
-  SortBy sort_by = 2;
+  oneof sort_by {
+    determined.checkpoint.v1.SortBy sortByEnum = 2;
```

Comment:
trying to think of a better name.. 'sort by enum' is strange, 'sort-by enum' makes more but i keep reading it the other way.
```suggestion
    determined.checkpoint.v1.SortBy sortByAttr = 2;
```

Reply:
Attr makes sense, done